lib/core/router/app_router.dart

```diff
@@ -120,6 +122,13 @@ final GoRouter appRouter = GoRouter(
         );
       },
     ),
+    GoRoute(
+      path: AppRoutes.userManagement,
+      name: AppRoutes.userManagement,
+      builder: (context, state) {
+        return const UserManagementScreen();
+      },
+    ),
```

Comment:
```suggestion
 GoRoute(
      path: AppRoutes.userManagement,
      name: AppRoutes.userManagement,
      builder: (context, state) {
        return const UserManagementScreen();
      },
    ),
```
Chỗ này làm chưa tốt lắm